docs/concepts.rst

```diff
@@ -742,6 +742,7 @@ while creating tasks:
 * ``all_done``: all parents are done with their execution
 * ``one_failed``: fires as soon as at least one parent has failed, it does not wait for all parents to be done
 * ``one_success``: fires as soon as at least one parent succeeds, it does not wait for all parents to be done
+* ``atleast_one_success``: fires if at least one parent succceeds, it waits for all parents to be done
```

Comment:
I think we should rename it to something like `one_access_wait_all`?  People might not be able to tell the difference between `atleast_one_success` and `one_success` without looking through the docs.